gulpfile.js

```diff
@@ -109,7 +110,7 @@ gulp.task('build:testjs', function () {
   gulp.src('src/*.js')
     .pipe(replace('{{apiurl}}', TESTAPIURL))
     .pipe(replace('{{cssurl}}', TESTCSSURL))
-    .pipe(replace('{{templates}}', prepareTemplates()))
+    .pipe(replace('\'{{templates}}\'', prepareTemplates()))
```

Comment:
this will work but it tells me we are linting the src file rather than the compiled file that includes templates and such. not sure we should be running it against the src files or changing the template defs. slightly worried that this would make it past tests accidentally since its going to be a string even if templates don't render. the other way will cause a panic.